src/main/java/org/gridsuite/network/map/model/TemporaryLimitData.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.network.map.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * @author David Braquart <david.braquart at rte-france.com>
+ */
+@Builder
+@Getter
+public class TemporaryLimitData {
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private String name;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private Integer acceptableDuration;
```

Comment:
Could not the name integrate the unit ?

Reply:
we never mention the unit in member names